### PR TITLE
build: fix webpack 5 support for extension

### PR DESCRIPTION
**Pull request by idoshamun**
### Describe what this PR does

Due to an old webpack plugin it was not possible to run `lerna bootstrap` in clean projects.
The old plugin didn't support webpack 5, I migrated to a different plugin to achieve the same result.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

**Review by sshanzel — APPROVED** [review]
Nice catch! 🚢

File: packages/extension/package.json [hunk 1]
```diff
@@ -101,8 +102,6 @@
     "mini-css-extract-plugin": "^1.6.2",
     "nock": "^13.2.4",
     "node-fetch": "^2.6.6",
-    "sass": "^1.53.0",
```
Comment (rebelchris) [inline packages/extension/package.json:104]: Not sure if the sass is even needed anymore.
But we can re-evaluate in another test case.

**Review by rebelchris — APPROVED** [review]
Looks great, thanks for this

File: packages/extension/package.json (hunk 1 above)
Comment (idoshamun) [reply packages/extension/package.json:104]: Not sure as well but I agree let's revisit it later :)

**Pull request merged**